lib/pyb.py

```diff
@@ -200,9 +202,6 @@ def __init__(self, pin):
         """
         pass
 
-    def read_timed(self):
-        pass
-
     def read_timed_stop(self):
```

Comment:
i am not a pyboard user, and do not follow updates/removals.
same as the above , it was definede twice ( incorrect and correct) 
i just removed the mistake as i think was clearly documented.

I you want me to review all prior code and double check them against code for a board I do not even own, then i will retract this pull request.

Reply:
I understand that you removed "read_timed" because it was defined a second time. This comment is about "read_timed_stop". I was asking why you didn't remove "read_timed_stop". I thought you might have good reason to keep it in and I was looking for that.